views/default/user.html

```diff
@@ -98,36 +98,51 @@ <h4 class="col-md-4">{{=T( request.args(0).replace('_',' ').capitalize() )}}</h4
+                                <div class="col-sm-offset-2">
+                                    <label for="auth_user_username" class="control-label">Username</label>
+                                    <input type="text" class="form-control col-xs-4" id="auth_user_username" name="username"
+                                           placeholder="Enter username" autofocus>
+                                </div>
                             </div>
-                        </div>
-                        <div class="form-group">
-                            <div class="col-sm-offset-2">
-                                <label for="auth_user_password" class="control-label">Password</label>
-                                <div>
-                                    <input type="password" class="form-control" id="auth_user_password" name="password"
-                                        placeholder="Enter password">
+                            <div class="form-group">
+                                <div class="col-sm-offset-2">
+                                    <label for="auth_user_password" class="control-label">Password</label>
+                                    <div>
+                                        <input type="password" class="form-control" id="auth_user_password" name="password"
+                                            placeholder="Enter password">
+                                    </div>
                                 </div>
                             </div>
-                        </div>
-                        <div class="form-group">
-                            <div class="col-sm-offset-2">
-                                <div class="checkbox">
-                                    <label>
-                                        <input id="auth_user_remember_me" name="remember_me" type="checkbox"> Remember me
-                                    </label>
+                            <div class="form-group">
+                                <div class="col-sm-offset-2">
+                                    <div class="checkbox">
+                                        <label>
+                                            <input id="auth_user_remember_me" name="remember_me" type="checkbox"> Remember me
+                                        </label>
+                                    </div>
                                 </div>
                             </div>
-                        </div>
-                        <div class="form-group">
-                            <div class="col-sm-offset-2 col-sm-4">
-                                <button type="submit" class="btn btn-default">Login</button>
+                            <div class="form-group">
+                                <div class="col-sm-offset-2 col-sm-4">
+                                    <button type="submit" class="btn btn-default">Login</button>
+                                </div>
                             </div>
```

Comment:
The github diff tool makes this look awful. It's actually just an added indent. Here's a snapshot from the SmartGit diff viewer that makes it clearer.

![image](https://user-images.githubusercontent.com/2125584/47391200-d33eeb80-d6de-11e8-90b1-d550ccdceaf9.png)